Common Functions/splice-functions/src/main/java/com/splicemachine/utils/RightPad.java

```diff
@@ -0,0 +1,36 @@
+package com.splicemachine.utils;
+
+public class RightPad {
+	
+	/**
+	 *  @author amitmudgal
+	 * RPAD function pads the right-side of a string with a specific set of characters (when string1 is not null).
+	 * "Lucubration", 10 
+	 * will return "Lucubratio".
+	 * "Lucubration", 15,'*'
+	 * will return "Lucubration****"
+	 * @param inboundString
+	 * @param paddedLength
+	 * @param padString 
+	 * @return  
+	 */
+
+	public static String rpad(String inboundString, int paddedLength, String padString) {
+		String paddedString = "";
```

Comment:
There is an existing utility package: org.apache.commons.lang3.StringUtils that implements the Pad logic for LeftPad/RightPad/Center/Repeat.  We should use that instead of reinventing the wheel.